autoload/vital/_iced/Data/Dict.vim

```diff
@@ -72,7 +72,7 @@ function! s:swap(dict) abort
   return s:make(values(a:dict), keys(a:dict))
 endfunction
 
-" Makes a index dict from a list
+" Makes an index dict from a list
```

Comment:
This file is installed by vital.vim.
So could you send a PR to the following repository?
https://github.com/vim-jp/vital.vim